app/actions/commute.js

```diff
@@ -0,0 +1,55 @@
+import _ from 'lodash';
+import directions from '../services/directions';
+
+export const updateCommutes = (dispatch, getState) => {
+  const state = getState();
+
+  const houseIds = Array.from(state.houses.keys());
+  const destinationIds = Array.from(state.destinations.keys());
+  const knownCommutes = Array.from(state.commutes.keys());
+
+  const desiredCommutes = destinationIds
+    .map((destinationId) => houseIds.map((houseId) => `${houseId}.${destinationId}`))
```

Comment:
if you're doing this just so you can split it later (line 22) it's prob preferred to make this an obj?

```
{ houseId, destinationId }
```

And destructure later?